packages/sdk/typescript/human-protocol-sdk/src/transaction.ts

```diff
@@ -172,6 +202,34 @@ export class TransactionUtils {
       return [];
     }
 
-    return transactions;
+    return transactions.map((transaction) => mapTransaction(transaction));
   }
 }
+
+function mapTransaction(t: TransactionData): ITransaction {
+  const internalTransactions: InternalTransaction[] = (
+    t.internalTransactions || []
+  ).map((itx) => ({
+    from: itx.from,
+    to: itx.to,
+    value: BigInt(itx.value || 0),
```

Comment:
```suggestion
    value: BigInt(itx.value),
```
Should it always have value?